sections/records.md

```diff
@@ -130,23 +137,42 @@ Records are where your data is stored. One record is stored per category, per ca
   "event_id": 1  
 }
 ```
-
-* This will return ```201 Created``` along with the current JSON representation of the record if the creation was a success.
-* ```week_reference``` will be determined based on the ```service_time_id``` and ```service_date_time```
+* This returns `201 Created` along with the current JSON representation of the record if creation succeeded
+* `week_reference` is determined using the submitted `service_time_id` and `service_date_time`
 
 ## Edit record
 
-* `PUT /records/1.json` will update the specified record
-
+* `PUT /records/1.json` updates the specified record
+* See the permitted fields for record payload in the [Editable fields](#editable-fields) section
+* Example payload:
 ```json
 {
   "campus_id": 2,
   "value": 20,
 }
 ```
-
-This will return ```200 OK``` if the update was a success, along with the current JSON representation of the record in the response body.
+* This returns `200 OK` if the update succeeded, along with the current JSON representation of the record in the response body
 
 ## Delete record
 
-* `DELETE /records/1.json` will delete the specified record and return ```204 No Content``` if that was successful. 
+* `DELETE /records/1.json` deletes the specified record and returns `204 No Content` if deletion succeeded
+
+## Editable fields
+
+The editable fields are:
+- `campus_id`
+- `category_id`
+- `end_time`
+- `end_week`
+- `event_id`
+- `replaces`
```

Comment:
This is assuming that https://github.com/lifechurch/churchmetrics/pull/654 is in effect. I guess may be that it's not yet deployed, but at least the PR has been merged into master, so I would say we're fine.